cellprofiler/gui/cpframe.py

```diff
@@ -1113,12 +1114,10 @@ def __on_close_all(self, event):
 
     @staticmethod
     def __on_new_cp(event):
-        import os
-
-        if not hasattr(sys, "frozen"):
-            os.system("open CellProfiler_python.command")
+        if hasattr(sys, "frozen"):
+            os.system("open -na /Applications/CellProfiler-{}.app".format(cellprofiler.__version__))
         else:
-            os.system("open -na CellProfiler.app")
+            os.system("cellprofiler")
```

Comment:
In case the person has multiple Pythons and/or CellProfiler's on their system, let's make the command here `python3 -m cellprofiler`